setup.cfg

```
@@ -113,7 +113,7 @@ install_requires =
     iso8601>=0.1.12
     # Logging is broken with itsdangerous > 2
     itsdangerous>=1.1.0, <2.0
-    jinja2>=2.10.1, <2.12.0
+    jinja2>=2.10.1
```

```suggestion
    jinja2>=2.10.1,<4
```

We don't know what Jinja4 might break